resources/aws_ec2.go

```diff
@@ -846,13 +865,30 @@ func (obj *AwsEc2Res) snsDeleteTopic(topicArn string) error {
 	return nil
 }
 
+// snsSubscribe subscribes the endpoint to the sns topic.
+// Returning SubscriptionArn here is useless as it is still pending confirmation.
```

Comment:
When is it no longer "pending confirmation" ? Do we need to wait for that to happen to be able to do something else?

Reply:
Lol, seems like never, but if you rerun confirm subscription after a little bit, it returns the arn. I don't really need it though, as deleting the topic deletes the subscriptions as well. I just thought I should explain why the method doesn't return anything.